package.json

```diff
@@ -6,10 +6,14 @@
   "browser": "dist/client.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rimraf dist && webpack-cli --mode production --config webpack.config.js",
+    "prepare": "yarn run build",
+    "build": "webpack-cli --mode production --config webpack.config.js",
```

Comment:
The previous `build` command cleaned the `dist` folder by running `rimraf dist`. Now, running `yarn build` will keep leftovers of all prior builds untouched, so they will most likely be pushed to npm.

Reply:
I have fixed it